### PR TITLE
Kops override artifact repos and remove PULL_NUMBER as release

### PR DESCRIPTION
**Pull request by TerryHowe**
Two things here:

1) Get rid of PULL_NUMBER the Prow environment variable from determining release. The calling script should set and export RELEASE
2) Allow overridiing the artifact web server and ECR repo.

### CODE REVIEW

File: development/kops/set_environment.sh [hunk 1]
```diff
@@ -70,6 +64,7 @@ export PREFLIGHT_CHECK_PASSED
 VERSION=$(cat ${BASEDIR}/../../projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG)
```
Comment (abhay-krishna) [inline development/kops/set_environment.sh:64]: Is this value being used in any script? Should it be exported in that case?

Comment (TerryHowe) [reply development/kops/set_environment.sh]: Done. It was working because VERSION was still in scope, but better to export for consistency

**Review by jaxesn — APPROVED** [review]
/lgtm

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**